api/queues.js

```diff
@@ -61,15 +65,18 @@ router.post(
       createdByUserId: res.locals.userAuthn.id,
     })
 
-    queue.save().then(newQueue => res.status(201).json(newQueue))
+    queue
+      .save()
+      .then(newQueue => res.status(201).json(newQueue))
+      .catch(next)
```

Comment:
why is this post catching instead of using the wrapper?

Reply:
The wrapper is really only for `async` stuff, this is using promises directly so it's better to use `catch` directly.